### PR TITLE
Building the nav

### PR DESCRIPTION
**Pull request by edleeks87**

### CODE REVIEW

File: src/components/containers/leaderboard/leaderboard.component.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      return type === 'asc' ? itemA - itemB : itemB - itemA;
+    });
+
+    const sortByString = (items, sortByValue, type) => items.sort((a, b) => {
+      const itemA = a[userPropMap[sortByValue]];
+      const itemB = b[userPropMap[sortByValue]];
+
+      if (type === 'asc') {
+        if (itemA < itemB) {
+          return -1;
```
Comment (Anthv96A) [inline src/components/containers/leaderboard/leaderboard.component.js:81]: We can one line this 
return (itemA == itemB) ? 0 : itemA > itemB ? 1 : -1

Comment (edleeks87) [reply src/components/containers/leaderboard/leaderboard.component.js:81]: Yeah I'll update and add lint ignore

File: src/components/containers/leaderboard/leaderboard.component.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      const y = b[userPropMap[sortByValue]];
+      return type === 'asc' ? x - y : y - x;
+    });
+
+    const sortByString = (items, sortByValue, type) => items.sort((a, b) => {
+      const x = a[userPropMap[sortByValue]];
+      const y = b[userPropMap[sortByValue]];
+
+      // eslint-disable-next-line
+      return type === 'asc' ? ((x < y) ? -1 : (x > y) ? 1 : 0) : ((x > y) ? -1 : (x < y) ? 1 : 0);
```
Comment (Anthv96A) [inline src/components/containers/leaderboard/leaderboard.component.js:83]: Gonna sound like a pain, but now this is a bit too convoluted to read. I meant something like this:

    if (type === 'asc') 
        return (itemA < itemB) ? -1 : (itemA > itemB)  ? 1 : 0

   return (itemA > itemB) ? -1 : (itemA < itemB) ? 1: 0
 
 if (type === 'desc') {
        if (itemA > itemB) {
          return -1;
        }

        if (itemA < itemB) {
          return 1;
        }
      }

File: src/components/containers/nav-bar/nav-bar.component.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      };
+    default:
+      return state;
+  }
+};
+
+const NavBar = ({ user, getUserAsync }) => {
+  const [state, dispatch] = useReducer(reducer, initialState); // kill
+
+  useEffect(() => { // kill
```
Comment (Anthv96A) [inline src/components/containers/nav-bar/nav-bar.component.js:43]: Kill this

Comment (Anthv96A) [reply src/components/containers/nav-bar/nav-bar.component.js:43]: In fact, don't bother actually. I'll take this stuff out once the PR merges

**Pull request merged**